steve/util.py

```diff
@@ -378,10 +378,14 @@ def out(*output, **kw):
     sys.stdout.write(output + '\n')
 
 
-def load_json_files(config):
+def load_json_files(config, path=None):
```

Comment:
We could calculate the value of the `rootdir` and put it in the config when we parse it out.